src/applications/vaos/appointment-list/components/cancel/CancelAppointmentSucceededModal.jsx

```diff
@@ -17,15 +17,13 @@ export default function CancelAppointmentSucceededModal({
         modalTitle={`Your ${typeText} has been canceled`}
         data-testid={`cancel-${typeText}-SuccessModal`}
         role="alertdialog"
+        primaryButtonText="Continue"
+        onPrimaryButtonClick={onClose}
+        uswds
```

Comment:
Replacing the button element to utilize `<VaModal>`'s `primaryButtonText` property and `onPrimaryButtonClick` custom event